buildspec/linuxIntegrationTests.yml

```diff
@@ -25,6 +25,9 @@ phases:
             - '>/dev/null pip3 install --upgrade aws-sam-cli'
             - '>/dev/null pip3 install --upgrade awscli'
             - '>/dev/null pip3 install pylint'
+            # Install Go
+            - '>/dev/null VERSION=$(curl "https://golang.org/VERSION?m=text") && wget -q "https://dl.google.com/go/$VERSION.linux-amd64.tar.gz"'
+            - '>/dev/null tar -C /usr/local -xzf $VERSION.linux-amd64.tar.gz && export PATH=$PATH:/usr/local/go/bin && export GOPROXY=direct'
```

Comment:
is `GOPROXY=direct` actually needed here? we do it explicitly elsewhere. Not sure if the environment from this "install" phase lives through the "build" phase; and if it does, then we wouldn't need the explicit `GOPROXY=direct` settings elsewhere, I guess.

Reply:
It's needed unfortunately. I'm not sure why but the environment variables are not being inherited consistently. Will need to investigate further as to why.